consensus/core/src/block_manager.rs

```diff
@@ -413,6 +533,9 @@ enum TryAcceptResult {
     // The block has been processed before and already exists in BlockManager (and is suspended) or
     // in DagState (so has been already accepted). No further processing has been done at this point.
     Processed,
+    // When a received block is <= gc_round, then we simply skip its processing as there is no meaning
+    // do any action on it or even store it.
+    Skipped,
```

Comment:
Actually initially I had it as `Rejected` - but then I considered `Skipped` as the rejection probably conveys a more strong message (ex because there is something wrong with the block). Here we just choose to not process it as there is no meaning to.